plugins/openweather/locales/locale_en.yml

```diff
@@ -0,0 +1,30 @@
+---
+en:
+    Openweather:
+        weather_reset: "Forcing API update to update weather."
+        default: "Everywhere Else"
+        condition: "The weather is %{condition}."
+        temperature: "It is %{temperature}%{degree}%{unit} with a humidity of %{humidity}% on this %{season} %{time_of_day}."
+        winds: "There are %{wind_speed} %{unit} winds from the %{wind_dir}."
+        
+# from https://www.campbellsci.com/blog/convert-wind-directions
+
+        wind_1: "north"
```

Comment:
May consider making this configurable like you did the weather descriptions.

Reply:
I"m on the fence on this one. The reason for this is to translate degrees into a direction and isn't much for any more added description.

Reply:
It's not a big deal either way.  I only really thought of it because I noticed you use two different spellings for "northwest" versus "south west" and thought folks might have their own preferences.  And it's conceivable that a fantasy/sci-fi setting might want to use  different descriptions instead of compass points.  Neither of those is necessarily a compelling reason to change it.

Reply:
I fixed my bad consistency in the local file